openedx/core/lib/api/authentication.py

```diff
@@ -81,3 +88,27 @@ def authenticate_credentials(self, request, access_token):
             raise AuthenticationFailed('Invalid token')
 
         return token.user, token
+
+
+class JwtAuthentication(JSONWebTokenAuthentication):
+    """
+    Custom authentication using JWT from the edx oidc provider.
```

Comment:
to avoid confusion, please remove "from the edx oidc provider" as that is not true in this case